docker-compose.yml

```diff
@@ -0,0 +1,62 @@
+version: '3'
+
+networks:
+    net:
+        name: net
+
+volumes:
+    aica:
+
+services:
+    db:
+        image: postgresql
+
+        build:
+            context: ./backend/docker
+            dockerfile: db.Dockerfile
+
+        # keep in sync with .env
+        ports:
+            - '5432:5432'
```

Comment:
Remove this, we don't need to expose the db port outside the internal "net" network